lavamoat/browserify/beta/policy.json

```diff
@@ -1669,7 +1669,8 @@
     "@metamask/remote-feature-flag-controller": {
       "packages": {
         "@metamask/base-controller": true,
-        "cockatiel": true
+        "cockatiel": true,
+        "uuid": true
```

Comment:
@MetaMask/policy-reviewers The policy change here is due to updating @metamask/remote-feature-flag-controller". It introduced a new `uuid` package when it didn't have this before.